css/style.css

```diff
@@ -0,0 +1,49 @@
+
+
+body{
+
+    background-color: #4044ca;
+    overflow: hidden;
+
+
+} 
+h1{
+    color:white;
+    font-size: 400%;
+    margin-top: 15vh;
+    margin-left: 20vw;
+   
+
+}
+img{
+    width: 40vw;
```

Comment:
- use px instead of vw
vw make it very large on destop view
- no need for margin, use 
 align-items: center;
 justify-content: center;
for its container instead